cogs/birthday/birthday.py

```diff
@@ -571,14 +595,12 @@ async def toggleSelfBirthday(self, ctx: Context):
         headerGood = f":white_check_mark: {bold(fnTitle)}"
 
         msgAllow = (
-            f"{headerGood}\n"
-            f":white_check_mark: Enabled. Members can set their birthdays themselves "
+            f"{headerGood}: "
+            f"{bold('Enabled')}. Members can set their birthdays themselves "
             f"{bold('ONCE')} and {bold('ONLY IF')} their birthdays were not already set."
         )
         msgNotAllow = (
-            f"{headerGood}\n"
-            ":negative_squared_cross_mark: Disabled. Members cannot set their birthdays "
-            "themselves."
+            f"{headerGood}: " f"{bold('Disabled')}. Members cannot set their birthdays themselves."
```

Comment:
nit: assuming this was another change caused by `black`.

Reply:
Yes.